worlds/civ_6/__init__.py

```diff
@@ -0,0 +1,192 @@
+import math
+import os
+from typing import Any, Dict, Optional, Set
+
+from .Data import get_boosts_data
+
+from .Rules import create_boost_rules
+from .Container import CivVIContainer, generate_goody_hut_sql, generate_new_items, generate_setup_file, generate_update_boosts_sql
+from .Enum import CivVICheckType
+from .Items import BOOSTSANITY_PROGRESSION_ITEMS, FILLER_DISTRIBUTION, CivVIItemData, FillerItemRarity, generate_item_table, CivVIItem, get_random_filler_by_rarity
+from .Locations import CivVILocation, CivVILocationData, EraType, generate_era_location_table, generate_flat_location_table
+from .Options import CivVIOptions
+from .Regions import create_regions
+from BaseClasses import Item, ItemClassification, MultiWorld, Tutorial
+from worlds.AutoWorld import World, WebWorld
+from worlds.LauncherComponents import Component, SuffixIdentifier, Type, components, launch_subprocess
+
+
+def run_client(url: Optional[str] = None):
+    print("Running Civ6 Client")
+    from .Civ6Client import main  # lazy import
+    launch_subprocess(main, name="Civ6Client")
+
+
+components.append(
+    Component("Civ6 Client", func=run_client, component_type=Type.CLIENT,
+              file_identifier=SuffixIdentifier(".apcivvi"))
+)
+
+
+class CivVIWeb(WebWorld):
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up Civlization VI for MultiWorld.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["hesto2"]
+    )]
+
+
+class CivVIWorld(World):
+    """
+    Civilization VI is a turn-based strategy video game in which one or more players compete alongside computer-controlled AI opponents to grow their individual civilization from a small tribe to control the entire planet across several periods of development.
+    """
+
+    game: str = "Civilization VI"

```

Comment:
```suggestion
    game = "Civilization VI"
```
`game` is already typed in the World class, so no need to type it here